### PR TITLE
Personal/masaadi/dotnet8 migartion iomt

### PR DESCRIPTION
**Pull request by maryamasaaditeh**

### CODE REVIEW

**Review by psapozh82 — DISMISSED** [review]
To me it seems like you're just changing .NET compilation level to 8 and updating packages which makes sense to me. I hope you tested the actual functional behavior of this code and that there's no regression.

I would also suggest getting another round of review from some team members who know more about this before merging.

**Comment by dustinburson** [comment]
> To me it seems like you're just changing .NET compilation level to 8 and updating packages which makes sense to me. I hope you tested the actual functional behavior of this code and that there's no regression.
> 
> I would also suggest getting another round of review from some team members who know more about this before merging.

Thanks @psapozh82.  Good point and this was done.  I had a similar ask on the prior PR that was abandoned.  We have been trying to trouble shoot the PR build trigger issues.

**Review by dustinburson — APPROVED** [review]

**Pull request merged**
